src/du/native/clonesize.c

```
@@ -186,10 +390,79 @@ static void rec_push(ThreadOut *t, int node, uint64_t alloc, uint64_t dlen, uint
+    }
+    t->crecs[t->ncrecs].fileid = fileid;
+    t->crecs[t->ncrecs].mtime_ns = mtime_ns;
+    t->crecs[t->ncrecs].dlen = dlen;
+    t->crecs[t->ncrecs].alloc = alloc;
+    t->crecs[t->ncrecs].ext_start = ext_start;
+    t->crecs[t->ncrecs].ext_count = ext_count;
+    t->ncrecs++;
+}
+
+/** Binary search the mmap'd cache. Returns NULL on miss or when the identity moved. */
+static const CacheEnt *cache_lookup(uint64_t fileid, uint64_t mtime_ns, uint64_t dlen, uint64_t alloc) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'cache_lookup' of similar type ('uint64_t') are easily swapped by mistake

(bugprone-easily-swappable-parameters)